src/test/java/andreeaG/tests/tema8/pageTests/RefreshPageTest.java

```diff
@@ -0,0 +1,20 @@
+package andreeaG.tests.tema8.pageTests;
+
+import andreeaG.tests.tema8.pageObjects.RefreshPage;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RefreshPageTest extends BaseTest {
+
+    @Test(groups = {"mobile", "desktop"})
+    public void refreshPageTest() {
+        driver = new ChromeDriver();
```

Comment:
Remove this line. We have BeforeMethods in BaseTest that create the driver instances.